lib/dom.mli

```diff
@@ -103,426 +103,1403 @@ val forwardRef : ('props -> domRef -> Core.element) -> 'props Core.component
+  val perspective : string -> style_key
+
+  val perspectiveOrigin : string -> style_key
+
+  val transform : string -> style_key
+
+  val transformOrigin : string -> style_key
+
+  val transformStyle : string -> style_key
+
+  val justifyItems : string -> style_key
+
+  val justifySelf : string -> style_key
+
+  val placeContent : string -> style_key
+
+  val placeItems : string -> style_key
+
+  val placeSelf : string -> style_key
+
+  val appearance : string -> style_key
+
+  val caret : string -> style_key
+
+  val caretAnimation : string -> style_key
+
+  val caretShape : string -> style_key
+
+  val userSelect : string -> style_key
+
+  val maxLines : string -> style_key
+
+  val marqueeDirection : string -> style_key
+
+  val marqueeLoop : string -> style_key
+
+  val marqueeSpeed : string -> style_key
+
+  val marqueeStyle : string -> style_key
+
+  val overflowStyle : string -> style_key
+
+  val rotation : string -> style_key
+
+  val rotationPoint : string -> style_key
+
+  val alignmentBaseline : string -> style_key
+
+  val baselineShift : string -> style_key
+
+  val clip : string -> style_key
+
+  val clipPath : string -> style_key
+
+  val clipRule : string -> style_key
+
+  val colorInterpolation : string -> style_key
+
+  val colorInterpolationFilters : string -> style_key
+
+  val colorProfile : string -> style_key
+
+  val colorRendering : string -> style_key
+
+  val cursor : string -> style_key
+
+  val dominantBaseline : string -> style_key
+
+  val fill : string -> style_key
+
+  val fillOpacity : string -> style_key
+
+  val fillRule : string -> style_key
+
+  val filter : string -> style_key
+
+  val floodColor : string -> style_key
+
+  val floodOpacity : string -> style_key
+
+  val glyphOrientationHorizontal : string -> style_key
+
+  val glyphOrientationVertical : string -> style_key
+
+  val imageRendering : string -> style_key
+
+  val kerning : string -> style_key
+
+  val lightingColor : string -> style_key
+
+  val markerEnd : string -> style_key
+
+  val markerMid : string -> style_key
+
+  val markerStart : string -> style_key
+
+  val pointerEvents : string -> style_key
+
+  val shapeRendering : string -> style_key
+
+  val stopColor : string -> style_key
+
+  val stopOpacity : string -> style_key
+
+  val stroke : string -> style_key
+
+  val strokeDasharray : string -> style_key
+
+  val strokeDashoffset : string -> style_key
+
+  val strokeLinecap : string -> style_key
+
+  val strokeLinejoin : string -> style_key
+
+  val strokeMiterlimit : string -> style_key
+
+  val strokeOpacity : string -> style_key
+
+  val strokeWidth : string -> style_key
+
+  val textAnchor : string -> style_key
+
+  val textRendering : string -> style_key
+
+  val rubyAlign : string -> style_key
+
+  val rubyMerge : string -> style_key
+
+  val rubyPosition : string -> style_key
+
+  [@@@js.start]
+
+  [@@@js.implem
+  type style_key = string * Js_of_ocaml.Js.Unsafe.any
+
+  type t = style
+
+  let string_style_key key value =
```

Comment:
And the "key" part of the declaration is called a "property", so it would perhaps be more accurate to call this `string_style_prop`, at least in the way it's being used, partially applied.